inspect/README.md

```diff
@@ -0,0 +1,195 @@
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.module.net`
+
+The scan api should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.block-by-number.hash`
+
+The hash of the block data retrieved from the scan API
+
+## Trace API Inspection
+
+### `trace-api.accessible`
+
+Checks trace APIs accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `trace-api.supported`
+
+Only necessary for Ethereum Mainnet.
+
+Returns `1` by default on other chains
+
+Checks trace APIs accessibility by establishing rpc connection and tracing the genesis block or the trace field in
+provided gRPC message
+
+`1` for accessible, `-1` for not
+
+### `trace-api.block-by-number.hash`
+
+The hash of the block data retrieved from the Trace API
+
+### `trace-api.trace-block.hash`
+
+The hash of the block trace data retrieved from the Trace API
+
+## Proxy Inspection
+
+### `proxy.accessible`
+
+Checks json-rpc proxy accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `proxy.chain-id`
+
+Chain ID of the provided json-rpc
+
+### `proxy.module.web3`
+
+The json-rpc proxy should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `proxy.module.eth`
+
+The json-rpc proxy should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `proxy.module.net`
+
+The json-rpc proxy should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `proxy.block-by-number.hash`
+
+The hash of the block data retrieved from the json-rpc proxy.
+
+### `proxy.history-support`
+
+Returns the earliest block height that proxy supports. Lower is better.
+
+## Network Inspection
+
+### `network.access.outbound`
+
+Tries to access https://api.forta.network
+
+`1` for ok, `-1` for not accessible
+
+### `network.speed.download`
+
+Runs a download performance benchmark
+
+Download speed in `mbps`
+
+### `network.speed.upload`
+
+Runs an upload performance benchmark
+
+Upload speed in `mbps`
+
+## System Resources Inspection
+
+### `resources.memory.total`
+
+Total memory in bytes
+
+### `resources.memory.available`
+
+Current available memory in bytes
+
+### `resources.storage.total`
+
+Total storage in bytes
+
+### `resources.storage.available`
+
+Current available storage in bytes
+
+### `resources.cpu.benchmark`
+
+The Time container needs to calculate first 1e5 prime numbers on average
+
+Average time needed in nanoseconds
```

Comment:
```suggestion
Average time in nanoseconds
```

Reply:
Oh, right. We do say average here. Don't mind the [comment above](https://github.com/forta-network/forta-core-go/pull/80#discussion_r929904555) - safe to remove "on average"